packages/fast-components/src/slider-label/slider-label.ts

```diff
@@ -0,0 +1,96 @@
+import { attr, FastElement, observable } from "@microsoft/fast-element";
+import { SliderConfiguration, SliderOrientation } from "../slider";
+import { Direction } from "@microsoft/fast-web-utilities";
+import { convertPixelToPercent } from "../slider/slider-utilities";
+
+const defaultConfig: SliderConfiguration = {
+    min: 0,
+    max: 0,
+    direction: Direction.ltr,
+    orientation: SliderOrientation.horizontal,
+};
+
+export class SliderLabel extends FastElement {
+    @observable
+    public positionStyle: string;
+
+    public root: HTMLDivElement;
+
+    @attr
+    public position: string;
+    private positionChanged(): void {
+        this.positionStyle = this.positionAsStyle();
+    }
+
+    @attr({ attribute: "hide-mark", mode: "boolean" })
+    public hideMark: boolean = false;
+    public config: SliderConfiguration = {
+        min: 0,
+        max: 0,
+        direction: Direction.ltr,
+        orientation: SliderOrientation.horizontal,
+    };
+
+    constructor() {
+        super();
+        this.getSliderConfiguration();
```

Comment:
This needs to be moved to the connected callback because someone can instantiate the control in JS and then later add it to the DOM.

Reply:
done